08_files_lab/01.py

```diff
@@ -1,34 +1,23 @@
-"""
-Write a program that takes 2 file names
-and appends the second file's contents to
-the end of the first one.
+#!/usr/bin/python
 
-So if file 'a.txt' has:
+import sys
+import os
 
-a1
-a2
+file1 = sys.argv[1]
+file2 = sys.argv[2]
 
-and file 'b.txt' has
 
-b1
-b2
-b3
+##bonus 1
+if (len(sys.argv[1])!=3):
```

Comment:
You don't need so many parenthesis here:

```
if len(sys.argv[1]) != 3:
  ...
```

Parenthesis are only required for function calls and to override operator precedence rules.